### ISSUE TITLE
`github create-pr`: check if base branch for PR exists in remote

### ISSUE THREAD

**Issue by rifel123** [PawelLipski/git-machete#335]
Attempt to create a pull request with base branch being already deleted from remote ends up with `Unprocessable Entity` error. (example in #332).

Proposed solution:
Perform `git fetch <remote>` at the beginning of `create-pr`. if base branch is not present in remote branches, perform `handle_untracked_branch` with relevant remote for missing base branch.

**Comment by PawelLipski** [comment]
Probably no longer needed after #336! since a reasonable message should show up to the user anyway

**Comment by PawelLipski** [comment]
Rel: #342

**Comment by rifel123** [comment]
Unfortunately messages from github API are not precise in every case. With missing base branch on remote we can expect  only this:
`{'message': 'Validation Failed', 'errors': [{'resource': 'PullRequest', 'field': 'base', 'code': 'invalid'}], 'documentation_url': 'https://docs.github.com/rest/reference/pulls#create-a-pull-request'}`

This kind of check is still needed.

**Comment by PawelLipski** [comment]
Whoops okay :/

**Comment by rifel123** [comment]
closed with #348
